src/test/unit/test_lumigo_utils.py

```diff
@@ -464,14 +469,15 @@ def asserting_extension(file_path, single):
 
 
 @pytest.mark.parametrize(
-    "errors, final_log", [(ValueError, "ERROR"), ([ValueError, Mock()], "INFO")]
+    "errors, final_log, host",
+    [(ValueError, "ERROR", "force_reconnect"), ([ValueError, Mock()], "INFO", "1.1.1.1")],
 )
-def test_report_json_retry(monkeypatch, reporter_mock, caplog, errors, final_log):
+def test_report_json_retry(monkeypatch, reporter_mock, caplog, errors, final_log, host):
     reporter_mock.side_effect = report_json
-    monkeypatch.setattr(Configuration, "host", "force_reconnect")
+    monkeypatch.setattr(Configuration, "host", host)
     monkeypatch.setattr(Configuration, "should_report", True)
     monkeypatch.setattr(http.client, "HTTPSConnection", Mock())
-    http.client.HTTPSConnection("force_reconnect").getresponse.side_effect = errors
+    http.client.HTTPSConnection(host).getresponse.side_effect = errors
```

Comment:
Not sure what you did here.
This test checks the retry mechanism in `report_json` - 
I.e. catch an error on the first try and perform retry.

I think that you need to create another test if you want to test your behavior